taxi/views.py

```diff
@@ -1,3 +1,20 @@
+from django.http import HttpResponse, HttpRequest
 from django.shortcuts import render
 
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code